webapp/app/templates/components.html

```diff
@@ -35,52 +35,64 @@
         {{ _field(field) }}
         <label class="checkmark" for="{{ field.name }}"></label>
-        {{ field_label(field, class="col-sm-10 col-form-label ml-3 pt-0") }}
+        {{ field_label(field, class="col-sm-10 col-form-label ml-3 pt-0", position_details_after=position_details_after) }}
     </div>
 {% endmacro %}
 
-{% macro consent_box(field, classes="") %}
+{% macro consent_box(field, classes="", position_details_after=False) %}
     <div class="form-row checkbox consent-box col-lg-10">
         {{ _field(field) }}
         <label class="checkmark" for="{{ field.name }}"></label>
-        {{ field_label(field, class="col-sm-10 col-form-label ml-3 pt-0") }}
+        {{ field_label(field, class="col-sm-10 col-form-label ml-3 pt-0", position_details_after=position_details_after) }}
     </div>
 {% endmacro %}
 
-{% macro radio_field(field) -%}
+{% macro radio_field(field, position_details_after=False) -%}
     <fieldset>
-        {{ field_label(field) }}
+        {{ field_label(field, position_details_after=position_details_after) }}
         {% for subfield in field %}
             <div>
                 {{ subfield(class="") }}
-                {{ field_label(subfield, class="col-sm-10 col-form-label") }}
+                {{ field_label(subfield, class="col-sm-10 col-form-label", position_details_after=position_details_after) }}
             </div>
         {% endfor %}
+        {%- if position_details_after %}
+            {%- set detail = field.render_kw['detail'] -%}
+            {%- if detail %}
+                {{ details(detail.title, [detail.text], details_id=field.id) }}
+            {% endif %}
+        {% endif %}
         {{ field_errors(field) }}
     </fieldset>
 {%- endmacro %}
 
-{% macro separated_field(field) -%}
+{% macro separated_field(field, position_details_after=False) -%}
     <fieldset id="{{ field.id }}">
-        {{ field_label(field) }}
+        {{ field_label(field, position_details_after=position_details_after) }}
         <div class="form-row-center">
             {{ _field(field) }}
         </div>
+        {%- if position_details_after %}
+            {%- set detail = field.render_kw['detail'] -%}
+            {%- if detail %}
+                {{ details(detail.title, [detail.text], details_id=field.id) }}
+            {% endif %}
+        {% endif %}
         {{ field_errors(field) }}
     </fieldset>
 {%- endmacro %}
 
-{% macro field_label(field, class="") %}
+{% macro field_label(field, class="", position_details_after=False) %}
     {%- set label = field.label -%}
     {%- set help = field.render_kw['help'] -%}
     {%- set detail = field.render_kw['detail'] -%}
 
-    {% if field.type == 'RadioField' %}
-        {% if label.text and not hide_label and not (field.render_kw and 'hide_label' in field.render_kw and field.render_kw["hide_label"]) %} {# Only show legend if there is a legend text to show #}
+    {% if field.type == 'RadioField' or field.type == 'UnlockCodeField' %}
```

Comment:
Not sure this is on purpose? If it's not on main, it probably shouldn't be here?

Reply:
What is this. You mean UnlockCodeField?

Reply:
This was from one of the merges

Reply:
Yes. I mean exactly that. Just thought, if it's not on main and you didn't change it on purpose, it's probably an error.

Reply:
Yes, if I am not totally wrong, it is on main and I do not know why GitHub is showing this change here.